README.md

```diff
@@ -20,7 +23,48 @@ And visit `localhost:3000` to preview your changes.
 
 ## Updating Navigation
 
-Site navigation is generated in TypeScript based on the section the user is currently browsing in. The navigation layout is managed in [`src/layouts/navigation.ts`](https://github.com/axelarnetwork/axelar-docs/blob/main/src/layouts/navigation.ts), with pages represented as a `title` and `href` in one of the `children` arrays. The site supports up to 3 levels of nested navigation, with the top level acting as navigation header. Each subsequent level is visually tabbed beneath its parent.
+Site navigation is generated in TypeScript based on the section the user is currently browsing in. The navigation layout is managed in [`src/content/_navigation/navigation.ts`](https://github.com/axelarnetwork/axelar-docs/blob/main/src/content/_navigation/navigation.ts), with pages represented as a `title` and `href` in one of the `children` arrays.
+
+### How to Update Each Top-Level Navigation Item (e.g., "Developers")
+
+1. Locate the Navigation Configuration File:
```

Comment:
I still don't fully understand this change. It seems like a huge shift and reorganization without a ton of benefit
1. navigation hierarchy MUST match the file hierarchy now, which might not always be desired
2. it takes a single navigation file that conditionally renders each section based on the user and splits them into separate files, reducing the flexibility of shared/ conditional content.